pro/cypress/e2e/step-definitions/signupJourneyCreateOfferer.cy.ts

```diff
@@ -0,0 +1,180 @@
+import { When, Then, Given } from '@badeball/cypress-cucumber-preprocessor'
+const siret = Math.random().toString().substring(2, 16)
+const offererName = 'MINISTERE DE LA CULTURE'
+
+beforeEach(() => {
+  cy.intercept({
+    method: 'GET',
+    url: '/venue-types',
+  }).as('venue-types')
+
+  cy.intercept('GET', `http://localhost:5001/sirene/siret/${siret}`, (req) =>
```

Comment:
Attention, il me semble qu'il y ait eu des modifications qui font que l'api n'est pas toujours sur localhost; on peut taper sur testing, normalement.

Reply:
Un truc comme ça devrait fonctionner

```suggestion
  cy.intercept('GET', `/sirene/siret/${siret}`, (req) =>
```

Reply:
OK, je modifie